network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -36,25 +138,19 @@ class ProductPage(FoundationMetadataPageMixin, Page):
     )
     review_date = models.DateField(
         help_text='Review date of this product',
+        null=True,
     )
     company = models.CharField(
         max_length=100,
         help_text='Name of Company',
         blank=True,
     )
-    # product_category = # TODO: Convert the ManyToMany field to an Orderable
-    # Keeping code commented out for quick reference in a future ticket
-    # models.ManyToManyField(
-    #     'buyersguide.BuyersGuideProductCategory',
-    #     related_name='pniproduct',
-    #     blank=True
-    # )
     blurb = models.TextField(
         max_length=5000,
         help_text='Description of the product',
         blank=True
     )
-    url = models.URLField(
+    product_url = models.URLField(
```

Comment:
as this will require template updating later, let's add a comment here that this rename requires template updating. Even if we get that done a few days from now.

Reply:
Good idea, thanks!